packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -852,3 +871,31 @@ class MediaQuery extends InheritedWidget {
     properties.add(DiagnosticsProperty<MediaQueryData>('data', data, showName: false));
   }
 }
+
+/// Describes the navigation mode to be set by a [MediaQuery] widget
+///
+/// The different modes indicate the type of navigation to be used in a widget
+/// subtree for those widgets sensitive to it.
+///
+/// Use `NavigationModality.of(context)` to determine the navigation mode in
+/// effect for the given context. Use a [NavigationModality] widget to set the
```

Comment:
update comment to remove reference to NavigationModality?

Reply:
Whoops.  Fixed.